packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -79,6 +79,11 @@ class TextStyleTween extends Tween<TextStyle> {
 
 /// An abstract widget for building widgets that gradually change their
 /// values over a period of time.
+///
+/// Subclasses' States must provide a way to visit the subclass's relevant
+/// fields to animate and [ImplicitlyAnimatedWidget] will automatically
```

Comment:
Maybe end the sentence after "animate" and have the rest be a new sentence? I had trouble parsing all the "and"s.